www/js/defaults.json

```diff
@@ -91,6 +104,8 @@
         "gurbaniTextColor": "#EAC445",
         "overlayLogo": true,
         "overlayLarivaar": false,
+        "translationFont": "muli",
+        "gurbaniFont": "gurbaniakhar",
```

Comment:
why not just keep it the same as font name? that would remove need to transform the value when reading/writing.

Reply:
Just wondering but can the bold ones be their own font family? The key is arbitrary so would it makes it easier for each option to correspond to its own font-face?